### ISSUE TITLE
Implement Timidity support

### ISSUE THREAD

**Issue by datajake1999** [stuerp/foo_midi#71]
I am working on a VSTi version of Timidity 0.2i, and have made the synth easier to integrate into other projects. Adding Timidity to foo_midi itself would allow the use of Gravis Ultrasound patches for MIDI playback. The repo for my VSTi can be found at https://github.com/datajake1999/Timidity_VSTi

**Comment by stuerp** [comment]
It should work out of the box. The VSTi is loaded by vsthost__.exe and the component communicates with it to render MIDI.

**Comment by stuerp** [comment]
Yeah, I got another report about the settings. I'll look into it. Not sure how it got broken.

**Comment by stuerp** [comment]
v2.16 has been released with a fix for the VSTi configuration not getting saved.
